proxy/http/HttpSM.h

```diff
@@ -473,7 +473,7 @@ class HttpSM : public Continuation
   void perform_cache_write_action();
   void perform_transform_cache_write_action();
   void perform_nca_cache_action();
-  void setup_blind_tunnel(bool send_response_hdr);
+  void setup_blind_tunnel(bool send_response_hdr, IOBufferReader *initial=0);
```

Comment:
I Have made the changes - need to run through tests to ensure no regression (even though it is extremely unlikely)